sql/hive/src/main/scala/org/apache/spark/sql/hive/execution/InsertIntoHiveTable.scala

```diff
@@ -43,8 +42,8 @@ import org.apache.spark.util.SerializableJobConf
 private[hive]
 case class InsertIntoHiveTable(
     table: MetastoreRelation,
-    partition: Map[String, Option[String]],
-    child: SparkPlan,
+    @transient partition: Map[String, Option[String]],
+    @transient child: SparkPlan,
```

Comment:
This is probably a general way how we pass the data into the executors, we'd better keep it unchange, except we found this reduce the performance dramatically. Even for that, we probably need to review all of the Spark Plan nodes.